nixos/modules/services/games/factorio.nix

```diff
@@ -49,8 +49,13 @@ in
         default = 34197;
         description = ''
           The port to which the service should bind.
-
-          This option will also open up the UDP port in the firewall configuration.
+        '';
+      };
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
Actually I'd even advocate for defaulting to false here. Imho the only module that should open ports opt-out is openssh. Maybe you could add a second commit defaulting to false, so we can backport only the first commit?

Reply:
Yeah that makes sense.